hydra-android-gradle-plugin/README.md

```diff
@@ -25,12 +26,17 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.pandora.hydra:hydra-android-gradle-plugin:2.0.+'
+        // If you are using Android Gradle Plug-in < 3.4, change the below versions to 1.7.5
+        classpath 'com.pandora.hydra:hydra-android-gradle-plugin:2.0.0'
+        classpath "com.pandora.hydra:hydra-gradle-plugin:2.0.0"
```

Comment:
1. Why do we need to apply 2 plugins? One for unit tests and one for Android?
2. You could use open end dependency 2.0.+

Reply:
So in the next example I give (for the more complicated scenario), I show how to apply Hydra to a multi-module project that consists of a mix of Android apps, libraries, and standard Java libraries. For _that_ scenario, you need to apply both. I'll correct this.

I'll adjust the dependency specifier as well